docs/desktop-wpf/migration/convert-project-from-net-framework.md

```diff
@@ -120,107 +127,107 @@ Once the NuGet packages are updated to recent versions, the `<PackageReference>`
+* You also need to add a `<GenerateAssemblyInfo>false</GenerateAssemblyInfo>` property to the new project file since the Bean Trader sample includes assembly-level attributes (like `[AssemblyTitle]`) in an AssemblyInfo.cs file. By default, new SDK-style projects will autogenerate these attributes based on properties in the csproj file. Because you don't want that to happen in this case (the autogenerated attributes would conflict with those from AssemblyInfo.cs), you disable the autogenerated attributes with `<GenerateAssemblyInfo>`.
 
-* Although *resx* files are automatically included as embedded resources, other `<Resource>` items like images are not. So, copy the `<Resource>` elements for embedding image and icon files. We can simplify the png references to a single line by using the new project file format's support for globbing patterns: `<Resource Include="**\*.png" />`.
+* Although *resx* files are automatically included as embedded resources, other `<Resource>` items like images are not. So, copy the `<Resource>` elements for embedding image and icon files. You can simplify the png references to a single line by using the new project file format's support for globbing patterns: `<Resource Include="**\*.png" />`.
 
-* Similarly, `<None>` items will be included automatically, but they will not be copied to the output directory, by default. Because the Bean Trader project includes a `<None>` item that *is* copied to the output directory (using `PreserveNewest` behaviors), we need to update the automatically populated `<None>` item for that file, like this:
+* Similarly, `<None>` items are included automatically, but they aren't be copied to the output directory, by default. Because the Bean Trader project includes a `<None>` item that *is* copied to the output directory (using `PreserveNewest` behaviors), you need to update the automatically populated `<None>` item for that file, like this:
 
   ```xml
   <None Update="BeanTrader.pfx">
     <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
   </None>
   ```
 
-* The Bean Trader sample includes a XAML file (Default.Accent.xaml) as `Content` (rather than as a `Page`) because themes and accents defined in this file are loaded from the file's XAML at runtime, rather than being embedded in the app itself. The new project system automatically includes this file as a `<Page>`, though, since it's a XAML file. So, we need to both remove the XAML file as a page (`<Page Remove="**\Default.Accent.xaml" />`) and add it as content:
+* The Bean Trader sample includes a XAML file (Default.Accent.xaml) as `Content` (rather than as a `Page`) because themes and accents defined in this file are loaded from the file's XAML at runtime, rather than being embedded in the app itself. The new project system automatically includes this file as a `<Page>`, though, since it's a XAML file. So, you need to both remove the XAML file as a page (`<Page Remove="**\Default.Accent.xaml" />`) and add it as content:
```

Comment:
```suggestion
* The Bean Trader sample includes a XAML file (*Default.Accent.xaml*) as `Content` (rather than as a `Page`) because themes and accents defined in this file are loaded from the file's XAML at runtime, rather than being embedded in the app itself. The new project system automatically includes this file as a `<Page>`, though, since it's a XAML file. So, you need to both remove the XAML file as a page (`<Page Remove="**\Default.Accent.xaml" />`) and add it as content:
```